docs/quick-start/index.mdx

```diff
@@ -29,7 +29,7 @@ Crawlee comes with three main crawler classes: <ApiLink to="cheerio-crawler/clas
 This is a plain HTTP crawler. It parses HTML using the [Cheerio](https://github.com/cheeriojs/cheerio) library and crawls the web using the specialized [got-scraping](https://github.com/apify/got-scraping) HTTP client which masks as a browser. It's very fast and efficient, but can't handle JavaScript rendering.
 
 ### PuppeteerCrawler
-This crawler uses a headless browser to crawl, controlled by the [Puppeteer](https://github.com/puppeteer/puppeteer) library. It can control Chromium or Chrome. Puppeteer is the de-facto standard in headless browser automation.
+This crawler uses a headless browser to crawl, controlled by the [Puppeteer](https://github.com/puppeteer/puppeteer) library. It can control Chromium or Chrome and has experimental support for Firefox. Puppeteer is the de-facto standard in headless browser automation.
```

Comment:
i am not sure if this is true, i believe with v3 we kinda disallowed this combination due to problems detecting the actual implementation from puppeteer (they removed some public API for that iirc)

https://github.com/apify/apify-ts/commit/3ac0650106d2bd86c84a40a85061f0ae045ff1f0#diff-cdc381a22f96c91a2eaf50c26afccb6326eaa77cdad9f4ac9c8ded97a37c9ffaR184

Reply:
Hmm, good point. We probably should not mention it if it does not work 100%.